### DISCUSSION TITLE
Export des données cantines publiées

### DISCUSSION THREAD

**Discussion by hfroot** [betagouv/ma-cantine#1001] [Retours utilisateur·rice]
En format CSV
A voir les données à inclure

**Comment by alemangui** [comment]
Peut-être utiliser de l'open data ? https://www.data.gouv.fr/fr/  On vient d'assister à un webinaire de l'équipe data.gouv ou iels expliquent comment mettre à dispo des données publiquement. Ça a l'air assez simple,suffit de créer un compte, décrire le dataset, et mettre l'endpoint ou le fichier à rendre disponible.

Je te rejoins, il faut spécifier les données qu'on veut inclure.

Éventuellement on pourrait aussi rendre public les données agrégés des régions/département (ce qui est visible dans /statistiques-regionales)

**Reply by hfroot** [reply]
Ah oui, j'ai fait qqch très similaire avec data.ademe.fr, c'était assez facile.
